cluster/ci/config/roles.yaml

```diff
@@ -30,6 +24,8 @@ items:
   - stevekuznetsov
   - jwforres
   - kargakis
+  - eparis
+  - enj
```

Comment:
?

Reply:
@enj is already in the ci-admins group in our cluster. I am not sure what's the scope of the group honestly. cc: @smarterclayton

Reply:
This group gives cluster level access to everything. I don't think Mo needs that.

Reply:
I was given access to help figure out some OAuth / RBAC issues.  You are welcome to scope that down as you see fit.